### PR TITLE
Fix triggering scan without engines flags does not trigger secret detection by default (AST-69529)

### PR DESCRIPTION
**Pull request by LeonardoLordelloFontes**
- Fixed triggering scan without engines flags does not trigger secret detection by default
- Fixed edge case where the SCS scan would start without any micro engine (when it shouldn't start at all)
- Updated unit tests and integration tests accordingly

### CODE REVIEW

File: internal/commands/scan.go [hunk 1]
```diff
@@ -106,6 +106,8 @@ const (
 	ScsSecretDetectionType      = "secret-detection"
 	ScsRepoRequiredMsg          = "SCS scan failed to start: Scorecard scan is missing required flags, please include in the ast-cli arguments: " +
 		"--scs-repo-url your_repo_url --scs-repo-token your_repo_token"
+	ScsRepoWarningMsg = "SCS scan warning: Unable to start Scorecard scan due to missing required flags, please include in the ast-cli arguments: " +
```
Comment (arturfalcao) [inline internal/commands/scan.go:109]: update to SSCS `ScsRepoWarningMsg` and `ScsRepoRequiredMsg`

**Review by arturfalcao — APPROVED** [review]

**Review by AlvoBen — APPROVED** [review]
